src/visualization/plot_recipes.jl

```diff
@@ -284,20 +296,197 @@ end
 end
 
 
-# Create a PlotData2D plot directly from a TrixiODESolution for convenience
+
+"""
+    PlotData1D
+
+Holds all relevant data for creating 1D plots of multiple solution variables and to visualize the
+mesh.
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
+struct PlotData1D{Coordinates, Data, VariableNames, Vertices} <:AbstractPlotData
+  x::Coordinates
+  data::Data
+  variable_names::VariableNames
+  mesh_vertices_x::Vertices
+end
+
+# Create a PlotData1D object.
+function PlotData1D(u, semi;
+                    solution_variables=cons2prim,
+                    grid_lines=true, max_supported_level=11, nvisnodes=nothing,
+                    slice_axis=:z, slice_axis_intercept=0)
```

Comment:
I think it would be nice to add docstrings to methods like this (analogously to `PlotData2D`).

Reply:
Agreed! Also, you can remove the lines 318 to 320, since the keyword arguments are yet unused, as far as I can tell.

Reply:
Thanks for your feedback.

I'm looking forward to make changes accordingly and use a different branch for the next time.